setup.py

```diff
@@ -41,6 +41,7 @@
                    'Programming Language :: Python :: 3.4',
                    'Programming Language :: Python :: 3.5',
                    'Programming Language :: Python :: 3.6',
```

Comment:
is there a reason why 
```
'Programming Language :: Python :: 3.8',
```
is not here but it is in `build.yml` to be tested with? Sorry I know your PR is for `python 3.9` 😆

Reply:
I'll throw it in there